### PR TITLE
Add conda environment file

### PR DESCRIPTION
**Pull request by manics**
Also updates readme with instructions for using conda, and adds a travis test.

See: https://github.com/IDR/idr-py/pull/9#issuecomment-369936801

### CODE REVIEW

**Comment by jburel** [comment]
in your initial example, all dependencies were capped

**Comment by manics** [comment]
I know, this seems to work though.

**Comment by will-moore** [comment]
$ conda env create -f environment-idr-omero53.yml -n idr-omero53

SpecNotFound: Invalid name, try the format: user/package

**Comment by will-moore** [comment]
```
>>> from idr import connection
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/idr/__init__.py", line 4, in <module>
    from attributes import attributes_by_attributes # NOQA
  File "/usr/local/lib/python2.7/site-packages/idr/attributes.py", line 4, in <module>
    import pandas
ImportError: No module named pandas
```

**Comment by jburel** [comment]
@will-moore:
``conda env create -f environment-idr-omero53.yml -n idr-omero53``  works as expected
then
```
from idr import connection
No handlers could be found for logger "omero.gateway"
>>>
```

**Comment by jburel** [comment]
Everything works
The only point (discussed in office) is if we want to add  a range for the depencies.
Note that there is no range specified in the existing requirements file

**Comment by manics** [comment]
@will-moore Could you try:
```
conda env remove -n idr-omero53
conda create -n idr-omero53
source activate idr-omero53
conda info
# Paste output
```
Then:
```
conda env update -f environment-idr-omero53.yml
conda list
python -c 'import idr'
```

**Comment by will-moore** [comment]
I think there's something wrong with my install, which I did via:
```
./Miniconda2-latest-MacOSX-x86_64.sh
```
I get...
```
$ conda env remove -n idr-omero53

EnvironmentLocationNotFound: Not a conda environment: /Users/wmoore/miniconda2/envs/idr-omero53
```

**Comment by manics** [comment]
I've mentioned this in https://github.com/idr-contrib/community/issues/7
Any further comments?

File: environment-idr-omero53.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - python-omero=5.3.3
+  - zeroc-ice=3.6.3
+  - ipython
+  - ipywidgets
+  - matplotlib
+  - pandas
+  - requests
+  - seaborn
+  - pip:
+    - idr-py==0.2.0
```
Comment (sbesson) [inline environment-idr-omero53.yml]: Does this mean we will also need to bump this version on every release for consumers?

Comment (manics) [reply environment-idr-omero53.yml]: Good point, I've added a commit to remove the version.

File: README.rst [hunk 2]
```diff
@@ -15,6 +15,17 @@ Requirements
...
  * Python 2.6+
 
+Installing with conda
+=====================
+
+This requires `Anaconda/Miniconda with Python 2.7 <https://conda.io/docs/user-guide/install/download.html>`_.
+
+::
+
+    conda env create -f environment-idr-omero53.yml -n idr-omero53
```
Comment (joshmoore) [inline README.rst]: The file defines a name. Either drop `-n idr-omero53` or drop from the file.

File: environment-idr-omero53.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# conda env create -f environment-idr-omero53.yml
+name: idr-omero53
```
Comment (joshmoore) [inline environment-idr-omero53.yml]: As above, either a name here or on the command-line. (My preference would be none in the file).

File: environment-idr-omero53.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# conda env create -f environment-idr-omero53.yml
```
Comment (joshmoore) [inline environment-idr-omero53.yml]: I'd vote for making this `environment.yml`

**Comment by manics** [comment]
Updated

**Comment by joshmoore** [comment]
Looks good to me. Anyone else? Otherwise, I'd like to have this in prod47.

**Comment by joshmoore** [comment]
Travis is green, merging. Since this is largely for jupyter-external consumers, I propose not bumping the version number, though happy to release an `0.2.1` if anyone would prefer.

**Pull request merged**